synapse/handlers/federation.py

```diff
@@ -486,7 +484,7 @@ async def try_backfill(domains: StrCollection) -> bool:
                     # If this succeeded then we probably already have the
                     # appropriate stuff.
                     # TODO: We can probably do something more intelligent here.
-                    return True
+                    return None
```

Comment:
Would that not just be

```suggestion
                    return
```